flow/beta/cli/config/config.go

```diff
@@ -138,6 +360,7 @@ func (a AccountKey) MarshalJSON() ([]byte, error) {
 	})
 }
 
+// Save configuration to a path file in json format
```

Comment:
The convention in Go is to write function doc comments as a complete sentence where the first word always refers to the funtion (even when it's a bit awkward). These comments are parsed by [godoc](https://blog.golang.org/godoc)  and used to autogenerate documentation, so it's best to follow the expected format.

```suggestion
// Save saves the configuration to the specified path file in JSON format.
```

Reply:
BTW, it's also conventional in Go doc comments to always capitalize acronyms (e.g. JSON instead of json)